src/RecoveryServices/RecoveryServices.SiteRecovery/Models/PSFabricDetails.cs

```diff
@@ -117,6 +117,14 @@ public ASRProcessServer(ProcessServer processServer)
             this.SystemLoadStatus = processServer.SystemLoadStatus;
             this.TotalMemoryInBytes = processServer.TotalMemoryInBytes;
             this.TotalSpaceInBytes = processServer.TotalSpaceInBytes;
+            this.Health = processServer.Health;
```

Comment:
and also update existing tests to test these new properties

Reply:
The infrastructure used in previous tests is not present anymore. To add the new properties, I need to add the new infra. Will take this up in separate PR.